src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,3 +1,43 @@
-<p>
-  instructor-help-page works!
-</p>
+<div>
+  <h1 class="color-orange"> Help for Instructors</h1>
+  <div>
+    <p>
+      Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
+      <br> If you are new to TEAMMATES, our <a href="/web/front/getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
```

Comment:
This routing works when called from `web/front/help/instructor` but not when called from `web/instructor/help`. Use `../getting-started` instead.
Also, use `routerLink` instead of `href`.

Reply:
Done